sql/src/test/java/io/crate/operation/InputFactoryTest.java

```diff
@@ -158,7 +159,7 @@ public void testCompiled() throws Exception {
         FunctionImplementation impl = (FunctionImplementation) f.get(expression);
         assertThat(impl.info(), is(function.info()));
 
-        FunctionImplementation uncompiled = expressions.functions().get(function.info().ident());
-        assertThat(uncompiled, not(sameInstance(impl)));
+        FunctionIdent ident = function.info().ident();
+        assertThat( expressions.functions().get(ident.name(), ident.argumentTypes()), not(sameInstance(impl)));
```

Comment:
would be nice if we keep the `uncompiled` local variable as this is making it clearer what this assertion is about